src/app/app.component.html

```diff
@@ -1,9 +1,39 @@
-<dx-toolbar id="toolbar" style="margin-bottom: 21px">
-  <dxi-item
-    widget="dxButton"
-    [options]="buttonOptions"
-    location="after"
-  ></dxi-item>
-</dx-toolbar>
+<div
+  class="main-container"
```

Comment:
here we can use ngClass and add the styling in the css file. we rarely define any styling in the html file, only as a last resort.
even better would be to move this styling in the corresponding component